### PR TITLE
Question Added in Javascript

### PR DESCRIPTION
**Pull request by himalay12345**
New Question added with no answer in present.

### CODE REVIEW

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
@himalay12345 Please complete this PR by adding the answer.

**Review by himalay12345 — COMMENTED** [review]
I made the require changes.

File: javascript/answers/kumar-sourav-q45.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <div>
+                <h4>
+                    Is the classes in javascript similar to the classes in the other programming languages? 
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong>
+               JavaScript classes introduced in ECMAScript 2015 is extremely similar to classes in C++ and Java in terms of syntax. But, it actually just provides a clearer and simpler syntax for the existing prototype-based inheritance, in which classes are considered as special functions.
```
Comment (dakshkhetan) [inline javascript/answers/kumar-sourav-q45.html:52]: Insert a code snippet to demonstrate on JavaScript classes.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]

**Comment by dakshkhetan** [comment]
@himalay12345  Any updates? This PR will be closed in 24 hours if no changes are made.

**Comment by dakshkhetan** [comment]
Closing due to no response.

**Pull request closed without merging**